src/libraries/System.Net.Primitives/src/System/Net/Cookie.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 #nullable enable
+#pragma warning disable CA1834 // Prefer StringBuilder.Append(char) over Append(string) where applicable
```

Comment:
I'm assuming it's ok to sprinkle these in our code as appropriate?

Reply:
Why are they necessary- why not change the code?

Reply:
Alright, I reconsidered and changed our code :). (I initially wasn't sure that changing the code here was worth the effort.)